### ISSUE TITLE
Backpacks are partly invisible when a slime is wearing them

### ISSUE THREAD

**Issue by beck-thompson** [space-wizards/space-station-14#33862]
## Description

Title! Because slimes are see through you should be able to see the back of the backpack but you can't. I'm not sure how to fix this exactly because I think the clothing layer is above the entity layer so it might be difficult...

**Reproduction**

Equip a backpack as a slime!

**Screenshots**

![image](https://github.com/user-attachments/assets/7b13a788-3672-49f6-9cad-6863551e3c97)

**Comment by Everturning** [comment]
it's hidden on purpose because if it was visible the backpack would layer over your entire outfit

**Comment by deltanedas** [comment]
would need every backpack to have 2 layers for front and back side

**Comment by beck-thompson** [comment]
> would need every backpack to have 2 layers for front and back side

Yeah its probably not worth it it just annoyed me so I made the issue 😆
